plugins/azure_blob_storage/help.md

```diff
@@ -0,0 +1,496 @@
+Example input:
+
+```
+{
+  "additional_headers": {
+    "x-ms-client-request-id":"some_request_id",
+    "x-ms-lease-id":"exa12_lease_id"
+  },
+  "blob_name": "my_old_blob",
+  "container_name": "example_container_name",
+  "snapshot_id": "2022-05-24T15:22:30.1616830Z",
+  "snapshots": "include",
+  "version_id": "2022-05-20T15:38:24.8240240Z"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|delete_type|string|True|Blob's delete type|
+|message|string|True|Deletion message|
+|success|boolean|True|Whether the action was successful or not|
+
+Example output:
+
+```
+{
+  "success": true,
+  "message": "Blob deletion was successfully submitted.",
+  "delete_type": "soft"
+}
+```
+
+#### Delete Container
+
+The Delete Container action marks the specified container for deletion. The container and any blobs contained within it are later deleted during garbage collection.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|additional_headers|object|{}|False|Additional headers to pass to the API request|None|{"x-ms-meta-Name":"StorageSample","x-ms-blob-public-access":"blob"}|
+|container_name|string|None|True|Name of the container to delete|None|my_container|
+
+Example input:
+
+```
+{
+  "additional_headers": {
+    "x-ms-meta-Name":"StorageSample",
+    "x-ms-blob-public-access":"blob"
+  },
+  "container_name": "my_container"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|message|string|False|Deletion result message|
+|success|boolean|True|Whether the action was successful or not|
+
+Example output:
+
+```
+{
+  "success": true,
+  "message": "Container deletion was successfully submitted."
+}
+```
+
+#### Get Blob
+
+This action is used to the Get Blob action reads or downloads a blob from the system, including its metadata and properties. You can also use Get Blob to read a snapshot.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|additional_headers|object|{}|False|Additional headers to pass to the API request|None|{"x-ms-client-request-id":"some_request_id"}|
+|blob_name|string|None|True|Name of the blob to retrieve|None|my_new_blob|
+|byte_to_string|boolean|None|False|Whether output data should be converte from bytes to string or not|None|True|
+|container_name|string|None|True|Name of the container|None|example_container_name|
+|snapshot_id|string|None|False|The snapshot parameter is an opaque DateTime value that, when present, specifies the blob snapshot to delete|None|2022-05-24 15:22:30.161683+00:00|
+|version_id|string|None|False|The versionid parameter is an opaque DateTime value that, when present, specifies the Version of the blob to delete|None|2022-05-20 15:38:24.824024+00:00|
```

Comment:
```suggestion
|snapshot_id|string|None|False|The snapshot parameter is an opaque DateTime value that, when present, specifies the blob snapshot to delete|None|2022-05-24 15:22:30.161683|
|version_id|string|None|False|The versionid parameter is an opaque DateTime value that, when present, specifies the Version of the blob to delete|None|2022-05-20 15:38:24.824024|
```

This will satisfy the plugin validator.